### ISSUE TITLE
Pool liquidity pricing

### ISSUE THREAD

**Issue by p0mvn** [osmosis-labs/sqs#115]
Context

Currently, frontend recomputes liquidity pricing directly client side which is not performant. We need to return USDC liquidity for each pool as part of the /pools query.

To enable this, we first had to implement SQS pricing (complete).

The scope of this effort is to refactor our data ingest pattern from chain to be more ETL-like where the transform step computes the liqudiites using the new SQS pricing functionality.

As part of that, instead of pushing the pools into Redis at the end of the block, we define RPCs within SQS.

The chain calls these RPCs at the end of the block upon extracting data and transferring it to SQS. Then, sqs transforms the data by instrumenting the pools with liquidity prices and pushes them to Redis.

This has a number of benefits:
more granular control of data ingest opening up a number of future data transformation opportunities
enables pool liquidity pricing, removing complexity from the client side

The work in progress is here:
https://github.com/osmosis-labs/sqs/pull/82
https://github.com/osmosis-labs/osmosis/tree/roman/wip-ingest-refactor-v23

## Acceptance Criteria
- data ingestion is refactored #111 
- tested
- instrumented
- alerts added
- incrementally deployed
- FE team notified of the change being available

**Comment by p0mvn** [comment]
Blocked on #111

**Comment by p0mvn** [comment]
Closing in favor of https://github.com/osmosis-labs/sqs/issues/127
